can_tools/scrapers/official/base.py

```diff
@@ -1212,3 +1213,52 @@ def get_dataset(self, body, url) -> str:
         """Accepts JSON body, and a url to post to the data studio batched URL"""
         rawJson = str(requests.post(url, json=body).content)
         return rawJson
+
+
+class ETagCacheMixin:
+    """Mixin class to add the ability to check whether a dataset has been updated since last viewed.
+
+    This is used in the `create_cached_flow_for_scraper` flow, to determine whether
+    to ingest the data (execute the fetch, normalize, and put methods).
+    """
+
+    cache_url: str
```

Comment:
We have no standard/universal attribute used to store the URL that the scrapers fetch the data, so I've added one for this class